backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc

```diff
@@ -1,25 +1,32 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="oadp-backing-up-applications-restic-doc"]
-= Backing up applications with Restic
+= Backing up applications with File System Backup: Kopia or Restic
 include::_attributes/common-attributes.adoc[]
 :context: backing-up-applications
 
 toc::[]
 
-If your cloud provider does not support snapshots or if your applications are on NFS data volumes, you can create backups by using Restic.
+OADP enables you to back up and restore Kubernetes volumes attached to pods from the file system of the volumes, called File System Backup (FSB) or Pod Volume Backup. The data movement is accomplished by using modules from open source backup tools, Restic and Kopia.
+
+If your cloud provider does not support snapshots, or if your applications are on NFS data volumes, you can create backups by using FSB.
 
 [NOTE]
 ====
 link:https://restic.net/[Restic] is installed by the OADP Operator by default.
 ====
 
-Restic integration with OADP provides a solution for backing up and restoring almost any type of Kubernetes volume. This integration is an addition to OADP’s capabilities, not a replacement for existing functionality.
+FSB integration with OADP provides a solution for backing up and restoring almost any type of Kubernetes volume. This integration is an addition to OADP’s capabilities, not a replacement for existing functionality.
 
-You back up Kubernetes resources, internal images, and persistent volumes with Restic by editing the `Backup` custom resource (CR).
+You back up Kubernetes resources, internal images, and persistent volumes with Kopia or Restic by editing the `Backup` custom resource (CR).
 
 You do not need to specify a snapshot location in the `DataProtectionApplication` CR.
 
+[NOTE]
+====
+In OADP version 1.3 and later, you can use either Kopia or Restic for backing up applications. In OADP version 1.2 and earlier, you can only use Restic for backing up applications.
+====
+
 [IMPORTANT]
 ====
 Restic does not support backing up `hostPath` volumes. For more information, see link:https://{velero-domain}/docs/v{velero-version}/restic/#limitations[additional Restic limitations].
```

Comment:
Please replace 'Restic' with 'File System Backup'

Also, the link is broken.

The correct link is: https://velero.io/docs/v1.12/file-system-backup/#limitations